doc/user/data/explain_plan_operators.yml

```diff
@@ -0,0 +1,223 @@
+operators:
```

Comment:
Just made the table data into a  yaml file. I combined the raw plan and optimized plan data -- but added a `plan_type` to distinguish between the two.